api/src/test/java/org/openmrs/module/dhisreport/api/importsummary/importsummariesTest.java

```diff
@@ -0,0 +1,101 @@
+package org.openmrs.module.dhisreport.api.importsummary;
+
+import static org.junit.Assert.*;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.bind.Unmarshaller;
+
+import org.junit.Test;
+import org.openmrs.module.dhisreport.api.importsummary.ImportConflict;
+import org.openmrs.module.dhisreport.api.importsummary.ImportCount;
+import org.openmrs.module.dhisreport.api.importsummary.ImportSummaries;
+import org.openmrs.module.dhisreport.api.importsummary.ImportSummary;
+
+public class importsummariesTest {
```

Comment:
How about renaming this class to ImportSummariesTest? By the way, did you ever get an opportunity to look at this? http://www.oracle.com/technetwork/java/codeconventions-135099.html